File: C/Cap03/c03ex03.c
// c03ex03.c

#include <stdio.h>

int main(void) {
  double HT, VH, PD, SB, TD, SL;

  printf("Entre quantidade de horas trabalhadas ..: ");
  scanf("%lf", &HT);
  while (getchar() != '\n');

  printf("Entre valor do salario-hora ............: ");
  scanf("%lf", &VH);
  while (getchar() != '\n');

  printf("Entre valor do percentual de desconto ..: ");
  scanf("%lf", &PD);
  while (getchar() != '\n');

  SB = HT * VH;
  TD = (PD / 100) * SB;
  SL = SB - TD;

  printf("\n");
  printf("Salario Bruto ....: %lf\n", SB);
  printf("Salario Liquido ..: %lf\n", SL);

  printf("\nTecle <Enter> para encerrar... ");
  while (getchar() != '\n');

  return 0;
}

File: LUA/Cap03/c03ex03.lua
-- c03ex03.lua

io.write("Entre quantidade de horas trabalhadas ..: ")
ht = tonumber(io.read())

io.write("Entre valor do salario-hora ............: ")
vh = tonumber(io.read())

io.write("Entre valor do percentual de desconto ..: ")
pd = tonumber(io.read())

sb = ht * vh
td = (pd / 100) * sb
sl = sb - td

print("\nSalario Bruto ....: " .. string.format("%.2f", sb))
print("Salario Liquido ..: " .. string.format("%.2f", sl))

io.write("\nTecle <Enter> para encerrar... ")
io.read()
